spec/javascripts/components/step-by-step-nav-spec.js

```diff
@@ -171,12 +171,14 @@ describe('A stepnav module', function () {
   })
 
   it('inserts a button into each step to show/hide content', function () {
-    var $titleButton = $element.find('.js-step-title-button')
+    var $titleButtons = $element[0].querySelectorAll('.js-step-title-button')
 
-    expect($titleButton).toHaveClass('gem-c-step-nav__button--title')
-    expect($titleButton).toHaveAttr('aria-expanded', 'false')
-    expect($titleButton[0]).toHaveAttr('aria-controls', 'step-panel-topic-step-one-1')
-    expect($titleButton[1]).toHaveAttr('aria-controls', 'step-panel-topic-step-two-1')
+    for(var i = 0; i < $titleButtons.length; i++) {
+      expect($titleButtons[i]).toHaveClass('gem-c-step-nav__button--title')
```

Comment:
Where does `toHaveClass` come from? I can't find any documentation on it as a vanilla JS function (but I could be looking in the wrong places).

Reply:
Ah brilliant, thanks, hadn't seen that.

Reply:
Basically @AshGDS and I divided the work between us and `toHaveClass` is a `jasmine-jquery` function that Ash will be replacing so it will be replaced when he raises his PR.